docs/auth/federated-auth.md

```diff
@@ -91,6 +91,41 @@ Ensure the "Google" sign-in provider is enabled on the [Firebase Console](https:
   }
   ```
 
+## Game Auth Provider
+
+Firebase is compatible with specific game provider on Android (Play Games Sign-In) and iOS (Game Center).
```

Comment:
If Game Center isn't supported yet, how about just calling the section "Play Games sign-in" and removing the iOS tab?

Reply:
I agree with you and I have made the requested change. I was expecting Game Center integration to be easier